cli/dcoscli/data/help/package.txt

```diff
@@ -8,17 +8,17 @@ Usage:
     dcos package describe <package-name>
                           [(--app [--render]) --cli --config --options=<file>]
                           [--package-version=<package-version>]
-    dcos package describe <package-name> --package-versions
+                          [--package-versions]
```

Comment:
👍 to alphabetizing

The problem here is that --package-versions is very different than the other flags under "describe", and doesn't interact with other options. Separating them helps a little (the real change is to make it a different command).